server/services/premisesService.ts

```diff
@@ -34,6 +34,15 @@ export default class PremisesService {
     return { name: premises.name, summaryList }
   }
 
+  async getPremisesRadioList(): Promise<Array<{ text: string; value: string }>> {
```

Comment:
I think having these items as a select is fine. As a rule I tend to go for radios for enum-like values, and selects for things that are likely to change, or are fetched from the database.

Reply:
Yeah that makes sense. 
The `reason`, `moveOnCategory` and `destinationProvider` are going to be fetch from the DB soon too, should they be selects? 
I think it would make sense if the `destinationProvider` was but I see the other two as enums

Reply:
Yeah, they're all enum-like I think. There's definitely a more clear-cut way of seperating enum-like and database-y, but I'm not sure how to express it 😆

Reply:
I think it's maybe if there's more than 10 options, go with a select, otherwise stick with a radio. I'll see if there's some better guidance.

Reply:
Another thing to possibly think about with this, do we maybe want to look at using https://alphagov.github.io/accessible-autocomplete/?

@beth-dixxon Would love to know your take on it 👍

Reply:
See I had in my head the opposite was true, that users struggle with selects when there are too many options. 
It's worth bearing in mind that (judging from the pre-prod data) that there will be quite a lot of options: 

  - Move on Category: 12, 
  - Reason: 15,
  - Destination Provider: 20+ 

I don't think it's something we need to decide now but I think it's something worth thinking about. Ideally we'd narrow down the options in so we don't have present such a long list but as we're replacing an existing form we don't really have that option. Agree that Beth's opinion would be super helpful here

Reply:
I think selects can be difficult for users to navigate through if there’s a lot of options, so if there were 20+ it feels like the Autocomplete would be the better option. 

The only concern would be a scenario where users weren't sure of the options included, however with these coming directly from Delius it feels like we could use this option as users should be familiar with the options in the list. We could also use guidance to support this. 

Could we use the autocomplete component to show all of the values (as a dropdown) which will allow users to both browse through the list and type ahead if they wanted to?